chapters/unitexpressions.tex

```diff
@@ -62,9 +62,10 @@ \section{The Syntax of Unit Expressions}\label{the-syntax-of-unit-expressions}
 \end{lstlisting}
 
 A \lstinline[language=grammar]!unit-symbol! is a non-empty string of \lstinline[language=grammar]!unit-char! characters.
-The set of allowed characters is currently not defined.
+The set of allowed characters is currently not defined, but it is recommended to only use \lstinline[language=grammar]!NON-DIGIT!.
```

Comment:
I find this too weak.  If we should ever be able to build a working solution for unit checking, there cannot be uncertainty about counts as a valid unit symbol.

Reply:
I think this would be a good topic the upcoming phone meeting.  It would also be interesting to get input from others interested in unit checking, such as @casella, @perost, @jespermattsson.

Reply:
Poll (unit should be SI-unit, this is primarily displayUnit):
1. No change: Hans, Martin
2. Recommended NON-DIGIT: Gerd, Hans
3. Strictly NON-DIGIT: Henrik, Christoff, Markus, Martin, Elena.
=> Strict